cdap-app-fabric/src/main/java/co/cask/cdap/app/DefaultAppConfigurer.java

```diff
@@ -64,7 +66,7 @@
 public class DefaultAppConfigurer extends DefaultPluginConfigurer implements ApplicationConfigurer {
   private final ArtifactRepository artifactRepository;
   private final PluginInstantiator pluginInstantiator;
-  private final Id.Artifact artifactId;
+  private final co.cask.cdap.proto.id.ArtifactId artifactId;
```

Comment:
the problem is that ArtifactId is a public class in the API that is there so that select an artifact for a plugin. We can probably clean it up in a later pass.